src/jquery.ime.js

```diff
@@ -531,7 +532,7 @@
 	 */
 	FormWidgetEntry.prototype.getTextBeforeSelection = function ( maxLength ) {
 		var element = this.$element.get( 0 );
-		return this.$element.val().substring(
+		return this.$element.val().slice(
```

Comment:
Automatic change by eslint, which requires slice() instead of substring() and substr().